.github/workflows/lint.yml

```diff
@@ -198,6 +198,7 @@ jobs:
             -g"-torch/csrc/jit/export.cpp"            \
             -g"-torch/csrc/jit/import.cpp"            \
             -g"-torch/csrc/jit/netdef_converter.cpp"  \
+            -g"-torch/csrc/onnx/init.cpp"             \
```

Comment:
@houseroad added this line to exclude init.cpp (which includes onnx_pb.h) to fix the linter error, as explained in the comment above (line 191-193)